apollo-router/src/spec/query.rs

```diff
@@ -1052,30 +1019,45 @@ impl Query {
     pub(crate) fn contains_error_path(
         &self,
         operation_name: Option<&str>,
-        subselection: Option<&str>,
-        response_path: Option<&Path>,
+        label: &Option<String>,
         path: &Path,
+        variables_set: i32,
```

Comment:
this is a bitset indicating, for all boolean variables used in `@defer`'s `$if` argument, which ones are active

Reply:
Should we document that we support defered conditions up to 32?

Reply:
It’s even (artificially) limited to 4 because of exponential time and space use:

https://github.com/apollographql/router/blob/76236d8f5cc4df72b8d6b16e14b68b506f7d3dc2/apollo-router/src/spec/query/subselections.rs#L27-L29